ui/src/shared/components/TableGraph.js

```diff
@@ -172,6 +175,7 @@ class TableGraph extends Component {
             columnNames={
               tableOptions ? tableOptions.columnNames : [TIME_COLUMN_DEFAULT]
             }
+            columnIndex={columnIndex}
```

Comment:
it doesnt look like columnIndex was defined before this. could this also have a more specific name? :)